app/src/main/java/com/example/composestudy/presentation/SearchImageUI.kt

```diff
@@ -0,0 +1,191 @@
+package com.example.composestudy.presentation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import coil.compose.rememberAsyncImagePainter
+import com.example.composestudy.R
+import com.example.composestudy.domain.entity.Document
+import org.orbitmvi.orbit.compose.collectAsState
+
+@Composable
+fun SearchImageUI(viewModel: SearchViewModel = viewModel()) {
+    var searchText by remember { mutableStateOf(TextFieldValue()) }
+    val state by viewModel.collectAsState()
+
+    Surface(color = Color.White) {
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.Top,
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            SearchBar(
+                searchText,
+                { query ->
+                    searchText = query
+                    viewModel.processAction(SearchAction.Search(query.text))
+                },
+                { searchText = TextFieldValue() },
+            )
```

Comment:
```suggestion
            SearchBar(
                searchText = searchText,
                onSearchTextChanged = { query ->
                    searchText = query
                    viewModel.processAction(SearchAction.Search(query.text))
                },
                onClearSearch = { searchText = TextFieldValue() },
            )
```
위 코드처럼 property name을 명시적으로 적어주면 어떨까요?